package.json

```diff
@@ -16,7 +16,10 @@
     "lint:js": "eslint .",
     "lint:css": "stylelint 'blocks/**/*.css' 'styles/*.css' 'templates/**/*.css'",
     "lint": "npm run lint:js && npm run lint:css",
-    "prepare": "husky install",
+    "lint:js:fix": "eslint . --fix",
+    "lint:css:fix": "stylelint 'blocks/**/*.css' 'styles/*.css' 'templates/**/*.css' --fix",
+    "lint:fix": "npm run lint:js:fix && npm run lint:css:fix",
+    "prepare": "husky",
```

Comment:
why `husky` instead of `husky install`?

Reply:
Since the version was updated this is the new way :)